code/game/objects/items/devices/desynchronizer.dm

```diff
@@ -0,0 +1,69 @@
+/obj/item/desynchronizer
+	name = "desynchronizer"
+	desc = "An experimental device that can temporarily desynchronize the user from spacetime, effectively making them disappear while it's active."
+	icon = 'icons/obj/device.dmi'
+	icon_state = "signmaker_forcefield"
+	item_state = "electronic"
+	w_class = WEIGHT_CLASS_SMALL
+	item_flags = NOBLUDGEON
+	lefthand_file = 'icons/mob/inhands/misc/devices_lefthand.dmi'
+	righthand_file = 'icons/mob/inhands/misc/devices_righthand.dmi'
+	materials = list(MAT_METAL=250, MAT_GLASS=500)
+	var/max_duration = 3000
+	var/duration = 300
+	var/obj/effect/abstract/sync_holder/sync_holder
+	
+/obj/item/desynchronizer/attack_self(mob/user)
+	if(!sync_holder)
+		desync(user)
+	else
+		resync()
+
+/obj/item/desynchronizer/examine(mob/user)
+	..()
+	to_chat(user, "<span class='notice'>Alt-click to customize the duration. Current duration: [duration / 10] seconds.</span>")
+	to_chat(user, "<span class='notice'>Can be used to .</span>")
+	
+/obj/item/desynchronizer/AltClick(mob/living/user)
+	if(!istype(user) || !user.canUseTopic(src, BE_CLOSE, ismonkey(user)))
+		return
+	var/new_duration = input(user, "Set the duration (5-300):", "Desynchronizer", duration / 10) as null|num
+	if(new_duration)
+		new_duration *= 10
+		new_duration = CLAMP(new_duration, 50, max_duration)
+		duration = new_duration
+		to_chat(user, "<span class='notice'>You set the duration to [duration / 10] seconds.</span>")
+
+/obj/item/desynchronizer/proc/desync(mob/user)
+	if(sync_holder)
+		return
+	sync_holder = new(get_turf(src))
```

Comment:
```suggestion
	sync_holder = new(drop_location())
```

Reply:
this would let you have a sync holder within a locker and whatever that would entail, although i don't know if it's really a problem